django_stomp/settings.py

```diff
@@ -19,3 +19,7 @@ def eval_settings_otherwise_raise_exception(
 STOMP_PROCESS_MSG_WORKERS = eval_settings_otherwise_raise_exception(
     "STOMP_PROCESS_MSG_WORKERS", eval_as_int_if_provided_value_is_not_none_otherwise_none
 )
+
+# headers that must be purged from messages if present
+# 'message-id': RabbitMQ uses it internally
+UNSAFE_OR_RESERVED_BROKER_HEADERS = ["message-id"]
```

Comment:
**thought (non-blocking)**: Do you think it would be useful to make it configurable using the project settings? Although, I don't find much use for a feature like that, it could help if someone don't wanna some header to be passed around.

Reply:
You are right, my friend! In terms of making it configurable, it doesn't make sense at all! However, I really looked for a centralized spot to put these headers that one could easily find out... now, what do you think about a static variable on the class Publisher?

Reply:
I like that approach! It's something that concerns to the `Publisher` only and not to the whole `django-stomp` project.

Really great!